release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -754,6 +754,48 @@ The tests must perform an environment configuration every time they are executed
 
 xref:../scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc#discovery-mode_cnf-master[Discovery mode] validates the functionality of a cluster without altering its configuration. Existing environment configurations are used for the tests. The tests attempt to find the configuration items needed and use those items to execute the tests. If resources needed to run a specific test are not found, the test is skipped, providing an appropriate message to the user. After the tests are finished, no cleanup of the pre-configured configuration items is done, and the test environment can be used immediately for another test run.
 
+[id="ocp-4-6-haproxy-version-bump"]
+==== HAProxy version upgrade
+
+Ingress in {product-title} {product-version} now uses HAProxy version 2.0.16.
+
+[id="ocp-4-6-x-forwarded-headers"]
+==== Control over X-Forwarded headers
+
+Control over X-Forwarded headers is now possible by setting the `forwardedHeaderPolicy` parameter.
+
+Application and configuration of X-forwarded headers on a per-route basis is now supported with the introduction of the `haproxy.router.openshift.io/set-forwarded-headers` route annotations.
+
+See xref:../networking/ingress-operator.adoc#nw-using-ingress-forwarded_configuring-ingress[Using X-Forwarded headers] for more information.
+
+[id="ocp-4-6-modify-route-path"]
+==== Modify route path
+
+Modifying route paths for incoming requests is now supported with the  `haproxy.router.openshift.io/rewrite-target` variable.
+
+See xref:../networking/routes/route-configuration.adoc#route-configuration[Route configuration] for more information.
+
+[id="ocp-4-6-ingress-termination-policies"]
+==== Ingress termination policies
+
+Termination policies can now be defined by using the `route.openshift.io/termination` annotation for Ingress objects.
+
+See xref:../networking/routes/route-configuration.adoc#nw-ingress-creating-a-route-via-an-ingress_route-configuration[Creating a route through an Ingress object] for more information.
+
+[id="ocp-4-6-ingress-nlb-aws"]
+==== Ingress Controller Network Load Balancer for AWS
+
+Configuration of an Ingress Controller Network Load Balancer (NLB) for new and existing AWS clusters is now supported.
+
+See xref:../networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-aws-network-load-balancer.adoc#configuring-ingress-cluster-traffic-aws-network-load-balancer[Configuring ingress cluster traffic on AWS using a Network Load Balancer] for more information.
+
+[id="ocp-4-6-ingress-route53"]
+==== Ingress Operator endpoint configure for AWS Route53
```

Comment:
Perhaps `configuration` in this context would be better than `configure`?

Reply:
Oops. That's what I actually meant to type. Thank you!